gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -1039,13 +1034,33 @@ def calculate_update_time_ranges(
 
     # CHECKS
 
+    def parsing_quality_check(self,dataset: xr.Dataset):
+        """
+        Master function containing quality checks general to all datasets
+        If successful passes without comment. If unsuccessful raises a descriptive error message.
+
+        Parameters
+        ----------
+        original_dataset : xr.Dataset
+            The final dataset to be parsed
+        """
+        # Aggressively assert that the data is in contiguous time order. Only valid if the dataset's time dimension is longer than 1 element
```

Comment:
We need to update this to explain that the times are checked to be in an order defined by the set